router/core/init_config_poller.go

```diff
@@ -139,7 +140,7 @@ func InitializeConfigPoller(r *Router, cdnProviders map[string]config.BaseStorag
 
 	configPoller := configpoller.New(r.graphApiToken,
 		configpoller.WithLogger(r.logger),
-		configpoller.WithPollInterval(r.routerConfigPollerConfig.PollInterval),
+		configpoller.WithConfigPolling(r.routerConfigPollerConfig.PollInterval, r.routerConfigPollerConfig.PollJitter),
```

Comment:
Sry for the confusion. If you would rename it to just `WithPolling` it would be excellent.